wscl-issues/proposed/test-not-program-error

```diff
@@ -0,0 +1,469 @@
+Issue:          TEST-NOT-PROGRAM-ERROR
+Forum:          Cleanup
+Category:       CLARIFICATION
+Status:         proposed
+Edit History:   03-Sep-21, Version 1 by Tarn W. Burton
+References:     ADJOIN, ASSOC, COUNT, FIND, INTERSECTION, NINTERSECTION,
+                MEMBER, MISMATCH, POSITION, RASSOC, REMOVE, DELETE,
+                REMOVE-DUPLICATES, DELETE-DUPLICATES, SEARCH,
+                SET-DIFFERENCE, NSET-DIFFERENCE, SET-EXCLUSIVE-OR,
+                NSET-EXCLUSIVE-OR, SUBLIS, NSUBLIS, SUBSETP, SUBST,
+                NSUBST, SUBSTITUTE, NSUBSTITUTE, TREE-EQUAL, UNION,
+                NUNION
+
```

Comment:
The X3J13 issue [TEST-NOT-IF-NOT](http://www.lispworks.com/documentation/HyperSpec/Issues/iss345_w.htm) should probably be referenced. That issue contains the passage

> The :TEST-NOT keywords are not only functionally unnecessary but also problematic because it's not clear what to do when both :TEST and :TEST-NOT are provided. 

in its problem description.

Reply:
Is adding something like "X3J13 issue TEST-NOT-IF-NOT " at the end of that list ok? Not really clear to me from looking at the other proposed issues if there is a specific referencing style.

Reply:
Yeah, sorry, I thought we could wait with more extensive documentation until later. In general, you can check [the original proposal format description](https://github.com/s-expressionists/wscl/blob/main/x3j13-issues/proposal-format) as well as X3J13 issues in that directory. In this case, a `Related Issues: TEST-NOT-IF-NOT` "header field" would be appropriate.

Reply:
Ok, sounds good. Pushed a fix. Thanks for the feedback!